.idea/.name

```diff
@@ -0,0 +1 @@
+Tutorial
```

Comment:
I agree it's odd, unless this is needed to run the tutorials.  If so, as mentioned in the other PR, setting up Android Studio should be an explicit step that is part of the tutorial and is documented.